PowerShellTools/Intellisense/Utilities.cs

```diff
@@ -14,52 +14,52 @@ namespace PowerShellTools.Intellisense
 {
     internal static class Utilities
     {
-        internal static bool IsGroupStart(char ch)
+        public static bool IsGroupStart(char ch)

```

Comment:
I should have set them as Public from the first. Whether a method in a internal static class should be internal or public depends on when we change the class to public, do we still want the class to be internal. And that answer to be is no.